source/common/upstream/health_checker_impl.h

```diff
@@ -11,7 +12,6 @@
 #include "common/stream_info/stream_info_impl.h"
 #include "common/upstream/health_checker_base_impl.h"
 
-#include "include/envoy/api/_virtual_includes/api_interface/envoy/api/api.h"
```

Comment:
It might be worth adding a check against `_virtual_includes` to `./tools/check_format.py`.

Reply:
Yeah, can you pick this up as a follow-up?

Reply:
I added a regex check there. Wording could probably be better though.